docs/api/event-brokers.rst

```diff
@@ -130,25 +137,20 @@ events. In this example we are going to use the `python-kafka <https://kafka-pyt
     $ pip install rasa[kafka]
```

Comment:
is still still necessary and valid?

Reply:
nope, it's no longer optional (it's always installed since poetry was introduced)